sdk/entra/WebJobs.Extensions.AuthenticationEvents/test/ActionParameters.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Azure.WebJobs.Host.Executors;
```

Comment:
The path for these should be `tests` - we're missing an "s" here.  :smile:

Reply:
Done.